CLDR-15924 items not picked up in pr2297

macchiati opened the pull request:
CLDR-15924

Mike, please look these over. I tried to get all the items, except that I wanted to make sure we wanted the prefix on formal, even for non-long items.

- [ ] This PR completes the ticket.

<!--
Thank you for your pull request.
Please see http://cldr.unicode.org/index/process for general
information on contributing to CLDR.

1. Make sure the ticket is filed at
https://unicode-org.atlassian.net/projects/CLDR/
2. Update the PR title and first line of this
message to include the ticket ID (CLDR-_____)
3. You will be automatically asked to sign the contributors’
license before the PR is accepted.
- sign: https://cla-assistant.io/unicode-org/cldr
- license: http://www.unicode.org/copyright.html#License
-->

AEApple commented:
Did you mean to add Mike as a reviewer?

mimckenna commented on common/main/si.xml:
```diff
@@ -9589,12 +9589,12 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 		<initialPattern type="initialSequence">{0} {1}</initialPattern>
 		<personName order="givenFirst" length="long" usage="referring" formality="formal">
-			<namePattern>{given} {given2} {surname} {surname2} {suffix}</namePattern>
+			<namePattern>{prefix} {given} {given2} {surname} {surname2} {suffix}</namePattern>
```
This change to add `{prefix}` should be down at lines 9650-9652 in `<personName order="surnameFirst" length="long" usage="referring" formality="formal">`

macchiati replied:
I just retracted this change, and added the {prefix} further down

mimckenna commented on common/main/ta.xml:
```diff
@@ -11105,12 +11105,12 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 		<initialPattern type="initialSequence">↑↑↑</initialPattern>
 		<personName order="givenFirst" length="long" usage="referring" formality="formal">
-			<namePattern>{given} {given2} {surname} {surname2} {suffix}</namePattern>
+			<namePattern>{prefix} {given} {given2} {surname} {surname2} {suffix}</namePattern>
```
This change to add `{prefix}` should be in lines 11166-11168 for `<personName order="surnameFirst" length="long" usage="referring" formality="formal"`

macchiati replied:
I just retracted this change, and added the {prefix} further down

macchiati commented:
I also added the @TECHPREVIEW in the dtd.

common/dtd/ldml.dtd
Added to all the elements

common/dtd/ldmlSupplemental.dtd
minor change from running GenerateDtd to ensure that the changes worked

tools/cldr-code/src/main/java/org/unicode/cldr/tool/ChartDtdDelta.java
added signs to the charts to show the tech preview status. (cleaned up the code a bit to do that)

tools/cldr-code/src/main/java/org/unicode/cldr/util/DtdData.java
added the new @TECHPREVIEW, and made it work.
Note: slight refactoring to add getElementThrowingIfNull to simplify logic

macchiati commented:
> Did you mean to add Mike as a reviewer?
I have to do it manually.

macchiati commented:
I need a review to merge.

richgillam reviewed (approved):
Rubber stamp.
